models/h2o_dai_german_credit/app_h2o_mojo_pipeline.py

```diff
@@ -0,0 +1,45 @@
+from certifai.model.sdk import SimpleModelWrapper
+import pandas as pd
+import numpy as np
+from numpy import nan
+import datatable as dt
+import daimojo.model
+
+model = daimojo.model("./pipeline.mojo")
+print(f"Loaded {model.uuid} from ./pipeline.mojo")
+
+def _get_prediction_class(preds):
+    if preds[0] > preds[1]:
+        return 1
+    return 2
+
+class GermanCredit(SimpleModelWrapper):
+    def predict(self, npinstances):
+        columns = ['checkingstatus', 
+            'duration',
+            'history',
+            'purpose',
+            'amount',
+            'savings',
+            'employ',
+            'installment',
+            'status',
+            'residence',
+            'property',
+            'age',
+            'otherplans',
+            'housing',
+            'cards',
+            'liable',
+            'job',
+            'telephone'
+        ]
```

Comment:
these do not match the german credit dataset

Reply:
I'll change the README to differentiate between the two datasets. H2O DAI models don't work with the original dataset without dropped columns